models/role.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Models names are always capitalized! Please consider correcting the name of this file.